### PR TITLE
fix handling of empty strings in template literals

### PR DESCRIPTION
**Pull request by ewlsh**
The fix in c506769547e372684463b7966a465b66c0c3ea28 and #39 were not entirely compatible since `resolvePossibleStringValues` was re-used. I use flat strings in most projects so stumbled upon this debugging `""` elsewhere. The test suite also seemed to miss this edge case.

#### Checklist

- [X] only relevant code is changed (make a diff before you submit the PR)
- [X] run tests `npm run test`
- [X] tests are included
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/i18next/.github/blob/master/CONTRIBUTING.md)

#### Checklist (for documentation change)

- [ ] only relevant documentation part is changed (make a diff before you submit the PR)
- [ ] motivation/reason is provided
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/i18next/.github/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/extractor/parsers/ast-visitors.ts [hunk 1]
```diff
@@ -758,8 +758,8 @@ export class ASTVisitors {
           const ordinalAttr = node.opening.attributes?.find(
             (attr) =>
               attr.type === 'JSXAttribute' &&
-            attr.name.type === 'Identifier' &&
-            attr.name.value === 'ordinal'
+              attr.name.type === 'Identifier' &&
```
Comment (ewlsh) [inline src/extractor/parsers/ast-visitors.ts:761]: I accidentally formatted this in an earlier MR... so fixing them :)

**Comment by adrai** [comment]
good catch!!!


included in v1.5.11
